rest/model/artifact.go

```diff
@@ -0,0 +1,83 @@
+package model
+
+import (
+	"github.com/evergreen-ci/evergreen/model/artifact"
+	"github.com/pkg/errors"
+)
+
+type APIFile struct {
+	Name           APIString `json:"name"`
+	Link           APIString `json:"url"`
+	Visibility     APIString `json:"visibility"`
+	IgnoreForFetch bool      `json:"ignore_for_fetch"`
+}
+
+type APIEntry struct {
+	TaskId          APIString `json:"task"`
```

Comment:
The JSON tag should be `task_id` for consistency with the other models.

Reply:
done